lib/stages/ReadGain.cpp

```diff
@@ -133,6 +140,26 @@ bool ReadGain::update_gains_frb_callback(nlohmann::json& json) {
     return true;
 }
 
+bool ReadGain::update_UT1_UTC_offset(nlohmann::json& json) {
```

Comment:
This function should be moved to the `BeamformingPhaseUpdate` base class files.   Since that's where this value is needed.  I don't think anything in ReadGains needs the value.

Reply:
Done!